modules/ROOT/pages/sync/adding-sync-to-your-app.adoc

```diff
@@ -43,37 +34,61 @@ link:https://github.com/apollographql[Apollo client] requires some configuration
 3. Create a file with a name that corresponds to your business objects and a `.graphql` extension
 4. Add all your queries/mutations/subscriptions to the `.graphql` file. Apollo will generate the appropriate Java classes with nested classes for reading from the network response. 
 
-For example if you create a file `graphql/org/aerogear/android/app/memeolist/graphql/myProject.graphql` with the content bellow:
+For example if you create a file `graphql/whateverfolder/somefile.graphql` with the content bellow:
```

Comment:
I'd rather use a realistic example instead of `whateverfolder`

Reply:
I agree with you, but I have asked a friend out of the project to review it and he asked me if the folder **needs** to be `org/jboss/...` so I decided to replace that. Maybe `com/example` makes it better